TESTING.md

```diff
@@ -46,19 +46,28 @@ plugin, create a mailing with a block of content. Then try each of the following
  3. Edit a paragraph. Press Ctrl-Shift-T and see a dialog. Enter a filter and pick a token using the keyboard. Observe the new token in the paragraph.
  4. Edit a heading or button. Ensure that the the token icon/dropdown/hotkey work as expected. Observe the new token in the heading or button.
 
-## Manual Tests: Images
+## Manual Tests: Images and Links
 
-Mosaico handles a few different kinds of images. To test these, create a
+Mosaico handles a few different kinds of images and links. To test these, create a
 mailing then:
 
- 1. Add blocks with images
+ 1. Add blocks
    1. Add a block which supports an image. Upload an image.
-   2. Add a block which supports an image. Re-use the previously uplaoded image.
-   3. Add a footer block which uses the built-in Twitter/Facebook icons
- 2. Test the mailing. Check:
-   1. The first block should show the uploaded image. The URL should be absolute.
-   2. The second block should show the same uploaded image. The URL should be absolute.
-   3. The footer block should show the Twitter/Facebook icons. The URL should be absolute.
+   2. Add a block which supports an image. Re-use the previously uploaded image.
+   3. Add a footer block which uses the built-in Twitter/Facebook icons.
+   4. In one of the blocks, add a hyperlink to an external page (eg `https://google.com`).
+   5. In one of the blocks, add a hyperlink to an internal page (eg `http://dmaster.l/user`).
+   6. In one of the blocks, add a hyperlink to using a token (eg `{action.forward}`)
+ 2. (Definition: The "Message Evaluation Procedure" is as follows. We will reference this in subsequent steps.)
```

Comment:
Can we break out the "Message Evaluation Procedure" into it's own section for clarity?

Reply:
Good point. It was feeling a little awkward to put a definition inline.